docs/self-managed/operate-deployment/operate-configuration.md

```diff
@@ -62,9 +62,9 @@ the feature can be found in [the multi-tenancy documentation](../concepts/multi-
 
 The following configuration is required to enable multi-tenancy in Operate:
 
-| Name                                 | Description                                         | Default value |
-| ------------------------------------ | --------------------------------------------------- | ------------- |
-| camunda.operate.multiTenancy.enabled | Activates the multi-tenancy feature within Operate. | false         |
+| YAML path                            | Environment variable                 | Description                                         | Default value |
+| ------------------------------------ | ------------------------------------ | --------------------------------------------------- | ------------- |
+| camunda.operate.multiTenancy.enabled | CAMUNDA_OPERATE_MULTITENANCY_ENABLED | Activates the multi-tenancy feature within Operate. | false         |
```

Comment:
:wrench: If we're keeping the table format aligned across components, the env var name should be included in the description. Similar to Tasklist, we already mention at the beginning of this page that Operate is a Spring Boot application, and the standard configuration mechanisms of a Spring Boot application can be used.

Reply:
> [...] we already mention at the beginning of this page that Operate is a Spring Boot application, and the standard configuration mechanisms of a Spring Boot application can be used

This implies that people _know_ what spring boot is, and how it works. Let's not force folks to learn this as they try to configure Camunda. My two cents.